test/integration/test_calls.py

```diff
@@ -118,62 +119,65 @@ def test_create_call(self):
             Validate a Create Call request with all optional parameters
         """
         time.sleep(self.TEST_SLEEP)
-        create_call_response: CreateCallResponse = self.calls_api_instance.create_call(BW_ACCOUNT_ID, self.createCallBody, _return_http_data_only=False)
+        create_call_response: ApiResponse = self.calls_api_instance.create_call_with_http_info(
+            BW_ACCOUNT_ID,
+            self.createCallBody
+        )
 
         #Adding the call to the self.callIdArray
-        self.callIdArray.append(create_call_response[0].call_id)
+        self.callIdArray.append(create_call_response.data.call_id)
 
-        assert_that(create_call_response[1], 201)
-        assert_that(create_call_response[0], has_properties(
+        assert_that(create_call_response.status_code, 201)
+        assert_that(create_call_response.data, has_properties(
             'call_id', instance_of(str),
             'account_id', BW_ACCOUNT_ID,
             'application_id', BW_VOICE_APPLICATION_ID,
             'to', USER_NUMBER,
-            '_from', BW_NUMBER
+            'var_from', BW_NUMBER
         ))
 
     def test_create_call_bad_request(self):
         """Validate a bad (400) request
         """
-        call_body = CreateCall(to="invalidNumberFormat", _from=BW_NUMBER, application_id=BW_VOICE_APPLICATION_ID, answer_url=BASE_CALLBACK_URL)
+        call_body = CreateCall(to="invalidNumberFormat", var_from=BW_NUMBER, application_id=BW_VOICE_APPLICATION_ID, answer_url=BASE_CALLBACK_URL)
         
         with self.assertRaises(ApiException) as context:
-            self.calls_api_instance.create_call(BW_ACCOUNT_ID, call_body, _return_http_data_only=False)
+            self.calls_api_instance.create_call_with_http_info(BW_ACCOUNT_ID, call_body)
             
         assert_that(context.exception.status, 400)
 
     def test_create_call_unauthorized(self) -> None:
         """Validate an unauthorized (401) request
         """
         with self.assertRaises(UnauthorizedException) as context:
-            self.unauthorized_api_instance.create_call(BW_ACCOUNT_ID, self.testCallBody, _return_http_data_only=False)
+            self.unauthorized_api_instance.create_call(BW_ACCOUNT_ID, self.testCallBody)
 
         self.assertApiException(context, UnauthorizedException, 401)
 
     def test_create_call_forbidden(self) -> None:
         """Validate a forbidden (403) request
         """
         with self.assertRaises(ForbiddenException) as context:
-            self.forbidden_api_instance.create_call(BW_ACCOUNT_ID, self.testCallBody, _return_http_data_only=False)
+            self.forbidden_api_instance.create_call(BW_ACCOUNT_ID, self.testCallBody)
 
         self.assertApiException(context, ForbiddenException, 403)
 
 
     def test_get_call_state(self):
```

Comment:
do we need to add logic here to deal with the high latency for this operation?

Reply:
all the tests are passing as is so i say leave it for now 🤷

Reply:
We want to migrate to mocking anyways right?

Reply:
![sad manteca](https://img.freepik.com/premium-vector/cute-sad-butter-character_464314-1549.jpg?w=2000)